css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
```

Comment:
Yeah sorry I wasn't clear.  What I want you to do is make the changes on this branch, then push them up.  That will update the PR to a state that I'm happy to merge it.